src/pg/sql/42_observatory_exploration.sql

```diff
@@ -0,0 +1,30 @@
+
+-- return a table that contains a string match based on input
+-- TODO: implement search for timespan
+
+CREATE OR REPLACE FUNCTION OBS_SearchTables(
+  search_term text,
+  time_span text DEFAULT '2009 - 2013'
+)
+RETURNS text[]
```

Comment:
I think this should return a table, with the first column being `tablename` and the second being `timespan`.  For the sake of completeness, it could just as well return all the other `obs_table` columns.